Sources/ATProtoKit/Utilities/Logging/Logging.swift

```diff
@@ -5,54 +5,95 @@
 //  Created by Christopher Jr Riley on 2024-04-04.
 //
 
-import Logging
-
-#if canImport(os)
+// Choose the logging library based on the
+// platform we are working with
+#if os(macOS) || os(iOS) || os(tvOS) || os(watchOS) || os(visionOS)
+// If the platform is based on an Apple-product
+// then we will use the default-provided Apple logger
 import os
+#endif
+// This library will be used regardless for the log levels
+import Logging
 
+// Define the ATLogHandler
+// The log handler will automatically choose
+// the correct logging library based on the framework
```

Comment:
Is this documentation? If so, I'd suggest using three slashes instead of two, so that DocC can recognize it and add it into the documentation. In general, I'd like for all properties, methods, structs, classes, and protocols to be documented, regardless of whether it's a `public`, `private`, `internal`, or some other access modifier. This is simply to make things easier to understand for anyone using the SDK or anyone that's trying to contribute.

```suggestion
/// Define the ATLogHandler
/// The log handler will automatically choose
/// the correct logging library based on the framework.
```

Reply:
Oh ok, I will make some adjustments. I was not aware that the three slashes was the  documentation format for Swift.